create_playlist.py

```diff
@@ -37,123 +39,129 @@ def get_youtube_client(self):
+        scopes = ["https://www.googleapis.com/auth/youtube.readonly"]
+        flow = google_auth_oauthlib.flow.InstalledAppFlow.from_client_secrets_file(
+            client_secrets_file, scopes)
         credentials = flow.run_console()
 
         # From YOUTUBE_API
-        youtube_client = googleapiclient.discovery.build(api_service_name, api_version, credentials=credentials)
+        youtube_client = googleapiclient.discovery.build(
+            api_service_name, api_version, credentials=credentials)
 
         return youtube_client
 
-   #STEP 2: grab like song on youtube // and create a dictionary of important song info 
-    def get_liked_videos(self):  #Perfect
+    # STEP 2: grab like song on youtube // and create a dictionary of important song info
+    def get_liked_videos(self):  # Perfect
         request = self.youtube_client.videos().list(
-            part ="snippet,contentDetails,statistics",
-            myRating = "like"
+            part="snippet,contentDetails,statistics",
+            myRating="like"
         )
-        response =request.execute
+        response = request.execute
 
-         # collect each video and get important info #loops through all song called item
-        for item in response ["items"]:
+        # collect each video and get important info #loops through all song called item
+        for item in response["items"]:
             video_title = item["snipped"]["title"]
-            youtube_url ="https://www.youtube.com/watch?v={}".format(item["id"])  #plug in URL HERE (not done)
+            # plug in URL HERE (not done)
+            youtube_url = "https://www.youtube.com/watch?v={}".format(
+                item["id"])
 
-                #use youtube_dl lib to collect the song name and artist
-            video = youtube_dl.YoutubeDL({}).extract_info(youtube_url, download=False)
+            # use youtube_dl lib to collect the song name and artist
+            video = youtube_dl.YoutubeDL({}).extract_info(
+                youtube_url, download=False)
 
             song_name = video["track"]
             artist = video["artist"]
- 
-                 #save imported information
-            self.all_song_info[video_title]= {
-                "youtube_url" : youtube_url,
-                "song_name" : song_name,
-                "artist" : artist,
-
-                #add the uri, easy to get song to put into Playlist
-                "spotify_uri":self.get_spotify_uri(song_name,artist) #call the function we wrote 
+
+            # save imported information
+            self.all_song_info[video_title] = {
+                "youtube_url": youtube_url,
+                "song_name": song_name,
+                "artist": artist,
+
+                # add the uri, easy to get song to put into Playlist
+                # call the function we wrote
+                "spotify_uri": self.get_spotify_uri(song_name, artist)
```

Comment:
The `CreatePlaylist` class has no `get_spotify_uri` method.

Reply:
I'm calling the method I wrote called "get_spotify_uri" down a couple of lines

Reply:
one says ur**l**, the other says ur**i**